targets/drive/mobile/fastlane/metadata/android/fr-FR/full_description.txt

```diff
@@ -1,11 +1,16 @@
-Cette application n'est compatible qu'avec la dernière version de Cozy (v3).
+éunit. Dans votre Cozy, vous mettez toutes vos données personnelles : photos, santé, bancaire, vacances, alors inaccessibles aux autres et aux entreprises.
+Dans sa version Bêta (qui remplace l’ancienne version V2), Cozy vous propose une première application sur mobile : Cozy Drive. `
 
-Cozy Cloud permet à chacun d’avoir son cloud personnel, respectueux de la vie privée. Avec Cozy Drive, accédez aux fichiers et aux photos stockés dans votre Cozy depuis n'importe quel téléphone/tablette.
 
-Avec Cozy Drive vous pouvez :
-    - Accéder aux fichiers présents sur votre cloud personnel
-    - Synchroniser automatiquement les photos de votre téléphone ou tablette
+Fonctionnalités de Cozy Drive  :
+• Sauvegardez les photos prises depuis votre mobile, elles seront accessibles depuis votre PC et tablette
+• Automatisez la synchronisation de vos documents et fichiers sur votre PC avec votre Cozy
```

Comment:
It seems unclear that installing Cozy Mobile let you access your files that you synchronize with Cozy Desktop installed on your computer.